tests/python_tests/test_llm_pipeline_static.py

```diff
@@ -52,12 +50,13 @@ def generate_chat_history(model_path, device, pipeline_config, questions):
 @pytest.mark.parametrize("generation_config", generation_configs)
 def test_generation_compare_with_stateful(generation_config):
     prompt = 'What is OpenVINO?'
-    model_path = read_model(get_models_list()[0])[1]
+    model_id, tmp_path = get_models_list()[0]
+    _, _, model_path = download_and_convert_model(model_id, tmp_path)
```

Comment:
`tmp_path` and `model_path` is different with `model_id`

can `get_models_list` return already a final path where model will be saved? in this case we don't have to deal with both paths in each test

Reply:
We should use `model_id` to download model in case of empty path. So, I can offer a one more function to download and covert the sample model